tests/lib/rook/upstream.py

```diff
@@ -47,15 +41,17 @@ def build(self):
 
         self.get_golang()
         logger.info("Compiling rook...")
-        execute(command="make -j BUILD_REGISTRY='rook-build' IMAGES='ceph'",
-                env={"PATH": f"{self.workspace.bin_dir}/go/bin:"
-                             f"{os.environ['PATH']}",
-                     "TMPDIR": self.workspace.tmp_dir,
-                     "GOCACHE": self.workspace.tmp_dir,
-                     "GOPATH": self.workspace.build_dir},
-                log_stderr=False,
-                logger_name=(
-                    "make -j BUILD_REGISTRY='rook-build' IMAGES='ceph'"))
+        self.workspace.execute(
+            command="make -j BUILD_REGISTRY='rook-build' IMAGES='ceph'",
```

Comment:
So the actual difference in 9ad931f is this should be `"make --directory='{builddir}/src/github.com/rook/rook' -j BUILD_REGISTRY='rook-build' IMAGES='ceph' build"`

Reply:
Done